src/languageserverinstance.jl

```diff
@@ -29,6 +29,8 @@ function send(message, server)
 end
 
 function Base.run(server::LanguageServerInstance)
+    io = open("/home/zac/jls.log", "a")
```

Comment:
Absolute path?

Reply:
I'll pull this change as we've got telemetry and potentially an upstream fix for the self-deleting logs